lib/jerboa/format/exceptions.ex

```diff
@@ -1,4 +1,18 @@
 defmodule Jerboa.Format.HeaderLengthError do
+  @moduledoc """
+
```

Comment:
The lines don't change the HTML documentation. I put it here because I can wrap w/ a line but w/out I can't.